### DISCUSSION TITLE
ネザー 禍壺の砦A 開発状況

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#76] [エリア・建築の完成度・開発状況]
**壺内部の砦について**
座標地点 ネザー 617 107 1208
- [x] 土台の製作
- [x] 建築に着手
- [x] 装飾に着手
- 要望
某クッパ城をイメージして砦を作ってほしい。

**Comment by anminmakura** [comment]
![617_107_1208](https://user-images.githubusercontent.com/39670701/96727025-900ed480-13ed-11eb-9807-812acd7cacff.png)

**Comment by anminmakura** [comment]
れんと氏が土台及び建築に着手

**Comment by anminmakura** [comment]
完成確認

**Comment by huyuyori** [comment]
![2021-06-06_07 52 01](https://user-images.githubusercontent.com/73624149/120907421-374e2c80-c69c-11eb-8be7-9a300b805759.png)
完成
